LenovoLegionToolkit.WPF/Controls/Automation/Steps/DisplayBrightnessAutomationStepControl.cs

```diff
@@ -0,0 +1,51 @@
+﻿using LenovoLegionToolkit.Lib.Automation.Steps;
+using LenovoLegionToolkit.Lib;
+using System.Windows;
+using System.Windows.Controls;
+using Wpf.Ui.Common;
+using TextBox = Wpf.Ui.Controls.TextBox;
+using System.Threading.Tasks;
+
+namespace LenovoLegionToolkit.WPF.Controls.Automation.Steps
+{
+    public class DisplayBrightnessAutomationStepControl : AbstractAutomationStepControl<DisplayBrightnessAutomationStep>
+    {
+        private readonly TextBox _brightness = new()
+        {
+            PlaceholderText = "Brightness value",
+            Width = 150,
+        };
+
+        private readonly StackPanel _stackPanel = new();
+
+        public DisplayBrightnessAutomationStepControl(DisplayBrightnessAutomationStep step) : base(step)
+        {
+            Icon = SymbolRegular.BrightnessHigh48;
+            Title = "Display brightness";
+            Subtitle = "Change display brightness of the built-in display.\n\nWARNING: This action will not run correctly,\nif internal display is off.";
+        }
+
+        public override IAutomationStep CreateAutomationStep() => new DisplayBrightnessAutomationStep(_brightness.Text);
+
+        protected override UIElement? GetCustomControl()
+        {
+            _brightness.TextChanged += (s, e) =>
+            {
+                if (_brightness.Text != AutomationStep.Brightness)
```

Comment:
How about using a `NumberBox` here? It will be easier to validate the input - user can type whatever here, which is not great.